build-index.sh

```diff
@@ -168,6 +174,12 @@ function list_dir() {
         fi
 	      diff=$(rfcdiff "https://tools.ietf.org/id/${file}.txt" "$this_githubio")
         td "$(a "$diff" 'diff with last submission' diff "$file")"
+        this_issue_label=$(./lib/extract-frontmatter.py ${file}.md github-issue-label)
```

Comment:
How about a fallback based on the draft name:

```
echo "${file}" | cut -d- -f 4- -
```

That is, for "draft-ietf-wg-short-name", this would produce "short-name" as a default.

Reply:
That's somewhat likely to result in links that don't work; I'd rather have a positive assertion of the label from the author.